ts/packages/cli/src/helpers/stores/TypeORM.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  AuroraMysqlStore,
+  AuroraPostgreSQLStore,
+  CockroachStore,
+  MSSQLStore,
+  MariaStore,
+  MySQLStore,
+  PostgreSQLStore,
+  SQLiteStore,
+} from '@configu/node';
+import { InitFunctionParameters, SchemeToInit } from './types';
+
+const STI = async ({ uri, parsedUri, userinfo, queryDict }: InitFunctionParameters) => {
+  const [username, password] = userinfo;
```

Comment:
@rannn505, the username, and password could potentially include characters that break the URI pattern. For example, the username may be an email address that breaks the URI because of the `@` char. How do you propose we handle this?

Reply:
An issue will be opened for this.